requirements-dev.lock.txt

```diff
@@ -1,21 +1,32 @@
-cachetools==4.2.4
-certifi==2021.10.8
-charset-normalizer==2.0.7
-cray-product-catalog == 1.6.0
+attrs==22.1.0
```

Comment:
Were we just missing some stuff here, so you decided to regenerate the locked requirements?

Reply:
Yeah, I just went ahead and regenerated the locked requirements since I bumped the version of `kubernetes` as well to be compatible with the `csm-api-client` library.